man/systemd.netdev.xml

```diff
@@ -1848,6 +1851,28 @@
     Linux Ethernet Bonding Driver HOWTO</ulink></para>
   </refsect1>
 
+  <refsect1>
+    <title>[Xfrm] Section Options</title>
+
+    <para>The <literal>[Xfrm]</literal> section accepts the following
+    keys:</para>
+
+    <variablelist class='network-directives'>
+      <varlistentry>
+        <term><varname>Device=</varname></term>
+        <listitem>
+          <para>Sets the underlying network interface. This can be used for hardware offloading. Defaults to "lo".</para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><varname>InterfaceId=</varname></term>
+        <listitem>
+          <para>Sets the ID/key of the xfrm interface which needs to be associated with a SA/policy. Valid range is 0-0xffffffff, defaults to 0.</para>
```

Comment:
you specify this as hexadecimal value, that suggests you can specify hex values here, please say explicitly what is accepted here.